src/theme/components/button.ts

```diff
@@ -0,0 +1,91 @@
+const Button = {
+  // style object for base or default style
+  baseStyle: {
+    borderRadius: "2px",
+    cursor: "pointer",
+    color: "ui.white",
+    justifyContent: "center",
+    padding: "var(--space-xs) var(--space-s)",
```

Comment:
This is something that is dependent on another PR, but should you replace the `space` CSS vars with the Chakra data objects before merging this?

Reply:
This is fixed in #648 so I'll leave it there to see the progression of the Chakra integration for this component.